config/default/kustomization.yaml

```diff
@@ -1,12 +1,12 @@
 # Adds namespace to all resources.
-namespace: ksan-system
+namespace: openshift-ksan-storage-operator
 
 # Value of this field is prepended to the
 # names of all resources, e.g. a deployment named
 # "wordpress" becomes "alices-wordpress".
 # Note that it should also match with the prefix (text before '-') of the namespace
 # field above.
-namePrefix: ksan-
+#namePrefix: ksan-
```

Comment:
This confuses me because it's set here: https://github.com/openshift/ksan-operator/pull/10/files#diff-14effe9ea1a3ddb6e419e627a3501a6c42aa2dcbc76803923e9aff2438e6ee34R14

Reply:
I think in this case we just don't want it in the default, we primarily want it as part of the manager configuration.